packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -325,6 +334,11 @@ const useDropdown = (): UseDropdownReturnValue => {
       setShouldIgnoreBlur(true);
     }
 
+    // disable closing the select on blur events if we are using a bottomsheet
+    if (bottomSheetContext?.hasBottomSheet) {
+      setShouldIgnoreBlur(true);
```

Comment:
you might not need this after above `shouldIgnoreBlur` condition check change right?